example/Example.js

```diff
@@ -46,11 +43,51 @@ class Example extends Component {
       });
     }
 
-    onInit(){
-      Countly.pushTokenType(Countly.messagingMode.DEVELOPMENT, "Channel Name", "Channel Description");
-      Countly.enableLogging();
-      Countly.init("https://trinisoft.count.ly", "f0b2ac6919f718a13821575db28c0e2971e05ec5");
+    onInit = async() => {
+      if(!await Countly.isInitialized()) {
+        /**
+         * Recommended settings for Countly initialisation
+         * 
+         * @function setLoggingEnabled enable countly internal debugging logs
```

Comment:
Add these comments, and the ones from the following sections, at the end of each line